hack/test

```diff
@@ -106,7 +106,7 @@ function e2e_pass {
 
 	build_flags=("-i") # cache package compilation data for faster repeated builds
 	for i in {1..2}; do
-		go test "./test/e2e/" ${build_flags[@]} -run "$E2E_TEST_SELECTOR" -timeout 30m --race \
+		go test -parallel=4 "./test/e2e/" ${build_flags[@]} -run "$E2E_TEST_SELECTOR" -timeout 30m --race \
```

Comment:
Should we also set the `parallel` flag for e2e_slow pass?
Although by default the value of parallel is set to the number of cpus available so it might be fine.

Reply:
We don't parallelize the tests in e2e_slow. So it is fine to not set it.